addons/killtracker/XEH_postInit.sqf

```diff
@@ -19,7 +19,7 @@
 if ((getText (missionconfigfile >> "CfgDebriefingSections" >> QUOTE(XADDON) >> "variable")) != QXGVAR(outputText)) exitWith {
     TRACE_1("no mission debriefing config",_this);
 };
-if (!(["ace_medical"] call EFUNC(common,isModLoaded))) exitWith {
+if (!(GETEGVAR(medical,enabled,false))) exitWith {
```

Comment:
```suggestion
if !(GETEGVAR(medical,enabled,false)) exitWith {
```

Reply:
https://ace3.acemod.org/wiki/development/coding-guidelines#55-brackets-around-code
> Any conditions in statements shall always be wrapped around brackets.

but `if !(` has 600+ results...

Reply:
Imo this should be an exception, as imo it makes it harder to read.